source/reference/minio-mc-admin/mc-admin-replicate.rst

```diff
@@ -0,0 +1,370 @@
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc admin replicate add minio1 minio2 minio3
+
+         The following command explands an existing site replication that includes peer site ``minio1`` to an additional peer site, ``minio5``.
+         ``minio5`` contains no data.
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc admin replicate add minio1 minio5
+
+      .. tab-item:: SYNTAX
+
+         The command has the following syntax:
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc [GLOBALFLAGS] admin replicate add      \
+                                        ALIAS1        \
+                                        ALIAS2        \
+                                        [ALIAS3 ...]
+
+   .. mc-cmd:: ALIAS
+      :required:
+
+      The :ref:`alias <alias>` of a MinIO deployment to include in site replication.
+
+      At least two MinIO deployment aliases are required to create a site replication. 
+      Only the first alias can have buckets or objects.
+      The first site can also be empty.
+
+      To expand an existing site replication to one more new replication sites, the first :ref:`alias <alias>` must be a peer site in the site replication set to expand.
+      Then include one or more additional :ref:`aliases <alias>` to add to the existing site replication.
+      The deployments to add must be empty.
+
+
+.. mc-cmd:: edit
+   :fullpath:
+
+   Modifies the endpoint used for an existing peer site participating in site replication.
+
+   .. tab-set::
+
+      .. tab-item:: EXAMPLE
+
+         .. code-block:: shell
+            :class: copyable
+
+            mc admin replicate edit                                                   \
+                               minio2                                                 \
+                               --deployment-id c1758167-4426-454f-9aae-5c3dfdf6df64   \
+                               --endpoint https://minio2:9000
+
+      .. tab-item:: SYNTAX
+
+         The command has the following syntax:
+
+         .. code-block:: shell
+
+            mc [GLOBALFLAGS] admin replicate edit                       \
+                                        ALIAS                           \
+                                        --deployment-id [deploymentID]  \
+                                        --endpoint [newEndpoint]
+    
+   .. mc-cmd:: ALIAS
+      :required:
+
+      The :ref:`alias <alias>` of the MinIO deployment.
+
+   .. mc-cmd:: --deployment-id
+      :required:
+
+      The unique id of the deployment to change.
+
+      The deployment ID can be found by running :mc-cmd:`mc admin replicate info <ALIAS>`
+
+   .. mc-cmd:: --endpoint
+      :required:
+      
+      The new endpoint or URL to associate with the peer site.
+
+.. mc-cmd:: remove
+   :fullpath:
+
+   Removes one or more sites from a site replication configuration.
```

Comment:
Thought: Should we add a warning here (not necessarily an admonition) that you cannot re-add the removed sites without first wiping all replication-eligible data (e.g. empty state).

Reply:
Added a new paragraph

```
Remember, if you intend to re-add the site to a site replication configuration in the future, 
it must be empty of :ref:`replicable data <minio-site-replication-what-replicates>`.
```